README.md

```diff
@@ -46,6 +49,34 @@ Prior vSphere 8 Update 1, the Supervisor Services are only available with Superv
 | External DNS Service - ExternalDNS | ❌ | ✅ |
 *\* The embedded Harbor Registry feature is still available and supported on vSphere 7 and onwards.*
 
+## Consumption Interface
+
+Provides the Local Consumption Interface (LCI) for Namespaces within vSphere Client. This also includes the Single Sign On (SSO) component required by the Cloud Consumption Interface (CCI) in Aria Automation within VMware Cloud Foundation.
+
+The minimum required version for using this interface is vSphere 8 Update 3.
```

Comment:
I suggest adding whose 'version' we are talking about here.

Reply:
You want to say "The minimum required vSphere version for using this interface is vSphere 8 Update 3." ?

Reply:
Yes, that seems okay to me.